modules/ROOT/pages/dataweave-types.adoc

```diff
@@ -13,7 +13,7 @@ The types that DataWeave provide are bundled into modules that also contain the
 
 Types in the `dw::Core` module (Core types) are available without having to import the Core module. Other modules need to be imported for their functions and types to be available.
 
-This topic provides an introduction to some DataWeave types, and descriptions of common patterns, including:
+This topic describes the values that are possible for each DataWeave type and how to create those values. It also describes common patterns, including:
```

Comment:
To avoid self-reference, can we reword this? Maybe something like...:

It is important for you to know the values...type, how to create..., and the common patterns used: